core/templates/dev/head/pages/topic_editor/TopicEditorNavbarDirective.js

```diff
@@ -58,7 +58,11 @@ oppia.directive('topicEditorNavbar', [
               function() {
                 $scope.topicRights.markTopicAsPublished();
                 TopicEditorStateService.setTopicRights($scope.topicRights);
-              });
+              }).then(function(){
```

Comment:
Linting nit, here and elsewhere: add space before {

Reply:
@arod0214 did you see this comment?

Reply:
Done